### DISCUSSION TITLE
[MongoDB Preview] Occasional fatal "tls handshake eof"

### DISCUSSION THREAD

**Discussion by bosconian-dynamics** [prisma/prisma#10752] [Q&A]
Hello!

I'm running a migration script which is using Prisma to re-shape and migrate around 30 GB of data from a local mongo collection to a Serverless MongoDB Atlas database. It's working for the most part, but every so often during an upsert Prisma throws the following error:

```
  Error occurred during query execution:
ConnectorError(ConnectorError { user_facing_error: None, kind: RawError { code: "unknown", message: "tls handshake eof" } })
    at Object.request (/home/gh-site-deploy/staging/source/node_modules/@prisma/client/runtime/index.js:39041:15)
    at migrateFromSyncer (/home/gh-site-deploy/staging/source/scripts/seed.ts:80:9)
```

Every subsequent upsert then also fails with the same error - or sometimes an "unexpected EOF". If I wait a minute then re-start the migration script, it often carries on happily.

I'm at a loss on what's causing this, and did reach out to MongoDB Atlas support but have received no response in a week thus far. I'd really appreciate any leads or suggestions anyone might have!

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
